ansible/roles/machine/setup/tasks/configuration.yml

```diff
@@ -4,8 +4,25 @@
     state: "{{ selinux_mode }}"
     policy: targeted
 
-- name: enable FreeIPA COPR repo
-  shell: "dnf copr enable -y {{ copr_repo }}"
+- name: disable distro repositories
+  shell: "dnf config-manager --set-disabled {{ item }}"
+  with_items:
+    - fedora
+    - updates
+    - updates-testing
 
-- name: disable updates-testing repository
-  shell: dnf config-manager --set-disabled updates-testing
+- name: enable selected repositories
+  shell: "dnf config-manager --set-enabled {{ item.key }}"
+  with_dict:
+    fedora: "{{ repo_fedora_enabled }}"
+    updates: "{{ repo_updates_enabled }}"
+    updates-testing: "{{ repo_updates_testing_enabled }}"
+  when: item.value
+
+- name: configure rawhide repository
+  shell: "dnf config-manager --set-{{ 'enabled' if repo_rawhide_enabled else 'disabled' }} rawhide"
```

Comment:
Isn't redundant check if `repo_rawhide_enabled` is true? IIUC in all situations where `fedora_version` is `"rawhide"`,`repo_rawhide_enabled` is also true. There is some situation where `fedora_version` is `"rawhide"` and the repo should not be enabled?

Reply:
1. I wanted to specifically enable it instead of relying on the defaults.
2. I wanted to have an option to disable it, if we want to do so in the future. (for whatever reason, we just one day decide -- hey, let's just freeze rawhide packages for a week)

If you don't agree, I'm not against removing it, but I think it limits our options for no reason.